casper/network.py

```diff
@@ -34,7 +38,10 @@ def view_initialization(self, view):
         for validator in self.validator_set:
             validator.receive_messages(messages)
 
-    def random_initialization(self):
-        """Generates starting messages for all validators with None as an estiamte."""
+    def collect_initial_messages(self):
```

Comment:
should likely be prefixed with `_`. Maybe could use a name change. The name is a bit misleading because if the network moves forward in time, this will no longer collect the initial messages.